From 18101ce9160f007eb90758463ebcc2301e7d46d7 Mon Sep 17 00:00:00 2001
From: Robby Cai <R63905@freescale.com>
Date: Tue, 17 Jul 2012 10:54:11 +0800
Subject: [PATCH 1751/2463] ENGR00217371: imx6sl_arm2: Add WDOG_B pad
 configuration

Add missing WDOG_B pad configuration. Default setting is GPIO function
which is not appropriate for WDOG_B generation.

Signed-off-by: Robby Cai <R63905@freescale.com>
---
 arch/arm/mach-mx6/board-mx6sl_arm2.h |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-mx6/board-mx6sl_arm2.h b/arch/arm/mach-mx6/board-mx6sl_arm2.h
index d720ab3..09a2116 100755
--- a/arch/arm/mach-mx6/board-mx6sl_arm2.h
+++ b/arch/arm/mach-mx6/board-mx6sl_arm2.h
@@ -153,6 +153,9 @@ static iomux_v3_cfg_t mx6sl_arm2_pads[] = {
 	MX6SL_PAD_KEY_ROW1__KPP_ROW_1,
 	MX6SL_PAD_KEY_ROW2__KPP_ROW_2,
 	MX6SL_PAD_KEY_ROW3__KPP_ROW_3,
+
+	/* WDOG */
+	MX6SL_PAD_WDOG_B__WDOG1_WDOG_B,
 };
 
 static iomux_v3_cfg_t mx6sl_arm2_epdc_enable_pads[] = {
-- 
1.7.7.4

